src/components/Configure/layout/ConditionalProxyLayout/hasDisqualifyingKey.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * given a list of keys and disqualifying keys,
+ * return true if any of the disqualifying keys are found in the list of keys
+ * @param keys
+ * @param disqualifyingKeys
+ * @returns
+ */
+export function hasDisqualifyingKey(keys: string[], disqualifyingKeys: string[]): boolean {
```

Comment:
This function can have a simpler name like `hasKeys` or `hasOverlap`. The fact that keys are disqualifying keys is not really relevant to the implementation this function, it just checks for whether the first set of strings contains something in the second set (in other words, whether the 2 arrays overlap/intersects). And if you rename it to be something more generic you can put it in `utils`, it might be useful for other components later.

Reply:
updated to be hasKeys